### DISCUSSION TITLE
Cercare Segnali airtag e altri blutooth device da un drone usando un BRIDGE/Gateway

### DISCUSSION THREAD

**Discussion by daniele777** [indomus/forum#2895] [Nuove idee, proposte, progetti e guide]
Ragazzi vorrei dotare il mio drone con un 
diapositivo che possa captare segnali tipo airtag  si potrebbe usare questo circuito ? BRIDGE/Gateway autonomo (con NodeMCU ESP32 ) con la guida che ho trovato da voi naturalmente con modifiche... Pensavo a questo progetto per trovare degli animaletti dotati di airtag o samsung tag ...che ne pensate?

Grazie

**Comment by 50494554524F** [comment]
certo, con esp32/esphome puoi intercettare i mac dei dispositivi bluetooth nelle vicinanze... 

https://esphome.io/components/esp32_ble_tracker

magari puoi anche usare qualche altra schedina più economica in termini di € e consumi rispetto alle nodemcu...

**Reply by daniele777** [reply]
Hai in mente un piccolo circuito ?
Sai che modello ?

**Reply by 50494554524F** [reply]
qua c'è la lista delle schedine compatibili

https://registry.platformio.org/platforms/platformio/espressif32/boards

devi cercare un po' perché onestamente non mi ricordo quali sono...

edit: o forse erano quelle con esphome/RP2040, bhò...

**Reply by daniele777** [reply]
Ma pensi che il mio cellulare possa agganciarsi al bridge ci vorrà anche un software per leggere ed interpretare i segnali tu che ne pensi.?

**Comment by daniele777** [comment]
Ho inserito il primo file yaml

      `esphome:
       name: trackerblu
  
        esp32_ble_tracker:

         esp32:
         board: nodemcu-32s
          framework:
           type: arduino

          # Enable logging
           logger:

            # Enable Home Assistant API
             api:
             password: "mypasswd"

              ota:
               password: "mypasswd"

              # Enable fallback hotspot (captive portal) in case wifi connection fails
               ap:
               ssid: "Trackerblu Fallback Hotspot"
               password: "CAINTqeJzk2Y"

               captive_portal:
    `
